lib/origen/utility/csv_data.rb

```diff
@@ -77,7 +77,8 @@ def extract_csv_data(options = {}) # :nodoc:
         field_values = []
         result = []
         line_no = 1
-        File.readlines("#{Origen.root}/#{@file}").each do |line|
+        file = Origen.file_handler.clean_path_to(@file)
+        File.readlines("#{@file}").each do |line|
           # Clean up line
```

Comment:
Doesn't look right to me, updates `file` but then goes on to reference `@file`...

Reply:
Thanks, copy typo.  Strange that it worked anyway for my 2 tests cases with the bug.  test cases : file in plugin, file in toplevel app. Fixed.